samples/fr/Premiers pas/index.html

```diff
@@ -14,23 +14,23 @@
         <h2>Suivez le guide !</h2>
         
         <!--
-            MADE WITH <3 AND JAVASCRIPT
+            FAIT AVEC <3 ET JAVASCRIPT
         -->
         
         <p>
             Bienvenue dans Brackets, un éditeur de code open source qui comprend et facilite la conception de sites web. Il s’agit d’un éditeur à la fois léger et puissant qui intègre des outils visuels directement dans son interface, de sorte que chaque opération peut devenir un véritable jeu d’enfant.
```

Comment:
This paragraph may be edited like - 
‘’ Bienvenue dans Brackets, un éditeur de code open source qui comprend et facilite la conception de sites web. Il s’agit d’un éditeur à la fois léger et puissant qui intègre des outils visuels directement dans son interface, de sorte que chaque opération peut devenir un véritable jeu d’enfant.”